packages/xchain-cosmos/__tests__/client.test.ts

```diff
@@ -1,66 +1,71 @@
+import { cosmosclient, proto } from '@cosmos-client/core'
+import { Coin } from '@cosmos-client/core/cjs/openapi/api'
```

Comment:
avoid  cjs and ems imports. import {proto} from core instead and use proto.cosmos.base.v1beta1.Coin